dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -219,4 +236,60 @@ public static String getFileName(MultipartFile multipartFile)
             return multipartFile.getName();
         }
     }
+
+    private List<DSpaceObject> constructDSpaceObjectList(Context context,
+                                                         HttpServletRequest request) throws IOException {
+        List<String> list = readFromRequest(request);
+
+        List<DSpaceObject> dSpaceObjects = new LinkedList<>();
+        for (String string : list) {
+            String uuid = string.substring(string.lastIndexOf('/') + 1);
+            try {
+                for (DSpaceObjectService dSpaceObjectService : dSpaceObjectServices) {
+                    DSpaceObject dSpaceObject = dSpaceObjectService.find(context, UUIDUtils.fromString(uuid));
+                    if (dSpaceObject != null) {
+                        dSpaceObjects.add(dSpaceObject);
+                        break;
+                    }
+                }
+            } catch (SQLException e) {
+                log.error(e.getMessage(), e);
```

Comment:
I wonder if we should have a better error message here.  It seems like a SQLException here means that one of the objects listed in the uri-list is not found.

Reply:
This has been updated